reference/README.md

```diff
@@ -52,9 +55,17 @@ servicing client requests.
 
 Only the files in the include/internal, include/porting, src/internal, and src/porting directories
 need to be modified. All other code in the src and include directories is platform independent and
-should not be modified.
+should *NOT* be modified.
+
+Vendors *MUST* implement code to call the TA-side functions: ta_open_session_handler,
+ta_close_session_handler, and ta_invoke_command_handler which defined in ta.h and implemented in
+ta.c.
+
+Vendors *MUST* implement code identified by ```TODO SoC Vendor``` in files in the include/porting and
```

Comment:
Recommend checking and updating the code as I saw at least one instance where the tag was TODO Soc Vendor.